lib/cadet_web/views/assessments_view.ex

```diff
@@ -21,6 +21,9 @@ defmodule CadetWeb.AssessmentsView do
       reading: :reading,
       status: &(&1.user_status || "not_attempted"),
       maxGrade: :max_grade,
+      maxXp: :max_xp,
+      xp: &(&1.xp || 0),
```

Comment:
I feel like `xp: :xp` will suffice since both `user_total_grade()` and `user_total_xp()` return default values of 0. Correct me if I'm wrong on this.

Reply:
Oh you're right.